cpp/tools/ZeroC.Ice.Slice.Tools.Cpp/ZeroC.Ice.Slice.Tools.Cpp.csproj

```diff
@@ -6,31 +6,14 @@
       <Nullable>enable</Nullable>
       <LangVersion>10.0</LangVersion>
 
-      <!-- Packaging properties -->
-
-      <!-- The version for the NuGet package. -->
-      <!-- Defaults to IcePackageVersion, which is set in ice.version.props imported above. -->
-      <PackageVersion>$(IcePackageVersion)</PackageVersion>
-      <Description>Provides tools to generate C# code from Slice definitions; includes support for MSBuild projects.</Description>
-      <DevelopmentDependency>true</DevelopmentDependency>
-      <PackageType>Dependency</PackageType>
-      <IncludeBuildOutput>false</IncludeBuildOutput>
-      <!-- Suppress NU5100 warning for the package. -->
-      <!-- The assembly 'tasks\ZeroC.Ice.Slice.Tools.CSharp.dll' is not inside the 'lib' folder -->
-      <!-- We do that on purpose we don't want projects to reference it, is only for MSBuild to load the tasks. -->
-      <NoWarn>NU5100</NoWarn>
-      <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
-      <PackageTags>Ice;RPC;MSBuild;Slice</PackageTags>
-      <PackageReadmeFile>README.md</PackageReadmeFile>
-
       <!-- Signing properties -->
       <TimeStampServer Condition="'$(SIGN_TIMESTAMPSERVER)' == ''">http://timestamp.digicert.com</TimeStampServer>
       <SignCommandSHA1 Condition="Exists('$(SIGN_CERTIFICATE)')">signtool sign /f "$(SIGN_CERTIFICATE)" /p $(SIGN_PASSWORD) /t $(TimeStampServer)</SignCommandSHA1>
       <SignCommandSHA256 Condition="Exists('$(SIGN_CERTIFICATE)')">signtool sign /f "$(SIGN_CERTIFICATE)" /p $(SIGN_PASSWORD) /tr $(TimeStampServer) /td sha256 /fd sha256 /as</SignCommandSHA256>
 
     </PropertyGroup>
     <ItemGroup>
-      <Compile Include="..\ZeroC.Ice.Slice.Tools\*.cs" />
+      <Compile Include="..\..\..\csharp\tools\ZeroC.Ice.Slice.Tools\Common\*.cs" />
```

Comment:
I find it a little weird that the common code lives in the csharp folder but it's not really a big deal.